src/cli.nim

```diff
@@ -107,6 +111,7 @@ func genHelpText: string =
         of optSyncExercise: "slug"
         of optSyncMode: "mode"
         of optSyncProbSpecsDir: "dir"
+        of optUuidNum: "int"
```

Comment:
That line sets the parameter name in the help message. My thinking is that the below is clearer with `<int>` than with `<num>`: 

```
Options for uuid:
  -n, --num <int>              Number of UUIDs to generate
```

because `<int>` better implies that a fraction of a UUID is meaningless. That is, 2.5, √2, π and 2 + 3i are all *numbers*, but we're trying to ask the user for a (positive) integer.

Do you think there's something better than `<int>`? And is there a better option name than `-n, --num`?

And I think it's okay to write `<int>`, even though we didn't write `<string>` in the options for sync. The values of `<slug>`, `<mode>` and `<dir>` can't be "any string" - they have to be a valid slug, mode or directory.
```
Options for sync:
  -e, --exercise <slug>        Only sync this exercise
  -c, --check                  Terminates with a non-zero exit code if one or more tests are missing. Doesn't update the tests
  -m, --mode <mode>            What to do with missing test cases. Allowed values: c[hoose], i[nclude], e[xclude]
  -p, --prob-specs-dir <dir>   Use this `problem-specifications` directory, rather than cloning temporarily
  -o, --offline                Do not check that the directory specified by `-p, --prob-specs-dir` is up-to-date
```

Trivia: in the current implementation, all of
```
configlet_v3 uuid -n 2.5
configlet_v3 uuid -n 2/7
configlet_v3 uuid -n 2.9999999999999999999999999999999999999999999999999999999999999999
```
just output 2 UUIDs without complaining.

Reply:
Aha, now I understand. It's fine